google-cloud-storage/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+### 1.18.0 / 2019-04-09
+
+* Add support for V4 signed URLs.
+  * Add version param to #signed_url.
+* Fix file path encoding for V2 signed URLs.
+  * Change CGI encoding to URI (percent) encoding.
```

Comment:
Do you want to mention that this fixed signed URLs with spaces in the file name?

Reply:
Thanks, done!